src/lxc/utils.c

```diff
@@ -693,15 +693,17 @@ int detect_shared_rootfs(void)
 
 bool switch_to_ns(pid_t pid, const char *ns)
 {
-	int fd, ret;
-	char nspath[PATH_MAX];
+	__do_close_prot_errno int fd = -EBADF;
+	int ret;
+	char nspath[STRLITERALLEN("/proc//ns/") + INTTYPE_TO_STRLEN(pid_t) +
+		    LXC_NAMESPACE_NAME_MAX];
 
 	/* Switch to new ns */
 	ret = snprintf(nspath, PATH_MAX, "/proc/%d/ns/%s", pid, ns);
 	if (ret < 0 || ret >= PATH_MAX)
```

Comment:
s/`PATH_MAX`/`sizeof(nspath)`/ in the 2 lines above should fix the ci error.

Reply:
Ah, good catch.